.storybook/webpack.config.js

```diff
@@ -0,0 +1,46 @@
+module.exports = ({ config }) => {
+  config.module.rules.push({
+    test: /\.(ts|tsx)$/,
+    use: [
+      {
+        loader: require.resolve("awesome-typescript-loader"),
```

Comment:
Why use `awesome-typescript-loader` here and `ts-loader` in `webpack.config`? `ts-loader doesn't work for storybook?

Reply:
Why `require.resolve`? Webpack is supposed to correctly resolve paths by itself, doesn't work?

Reply:
Errr, not sure, I copied that from https://github.com/gnosis/safe-react-components/blob/development/.storybook/webpack.config.js

Will adjust

Reply:
f2cc4e1384f3d38031c830b8cfb5215c301c8bb4